app/views/idv/doc_auth/document_capture.html.erb

```diff
@@ -31,53 +29,61 @@
     ) do |f| %>
       <%# ---- Front Image ----- %>
 
-      <hr class="mt3 mb3"/>
-      <div class="front-image">
-        <h2>
-          <%= t('doc_auth.headings.upload_front_html') %>
-        </h2>
-
-        <%= accordion('totp-info', t('doc_auth.tips.title_html'),
-          wrapper_css: 'my2 col-12 fs-16p') do %>
-          <%= render 'idv/doc_auth/tips_and_sample' %>
-          <div class='center'>
-            <%= image_tag(asset_url('state-id-sample-front.jpg'), height: 338, width: 450) %>
-          </div>
-        <% end %>
+      <div><%= t('doc_auth.tips.document_capture_header_text') %></div>
```

Comment:
In #3990, I used a paragraph element here, based on the design specifications. I'm not positive if semantically-speaking there is a best way to express this sort of leading paragraph to a list. At least looking at the markup of ["Use lists" plain language guidance](https://plainlanguage.gov/guidelines/organize/use-lists/), a similar example there uses `<p>` as well.

```suggestion
      <p><%= t('doc_auth.tips.document_capture_header_text') %></p>
```

In #3990 and in talking with @anniehirshman-gsa , I found there's some fiddling that needs to be done to coerce the margin to something more like in the design. I used `margin-top-2 margin-bottom-0` to create the `1rem` spacing between the heading, and to bring the list closer to the leading paragraph.

```suggestion
      <p class="margin-top-2 margin-bottom-0"><%= t('doc_auth.tips.document_capture_header_text') %></p>
```

Reply:
Good call -- btw, you can abbreviate the classes: `margin-top-2 == mt2`, etc.

Reply:
> btw, you can abbreviate the classes: `margin-top-2 == mt2`, etc.

If I've understood correctly, they're both available and equivalent, but the former is from [USWDS](https://designsystem.digital.gov/utilities/margin-and-padding/) and the latter from [Basscss](https://basscss.com/#basscss-margin).